src/Components/Users/UserAdd.tsx

```diff
@@ -684,15 +683,12 @@ export const UserAdd = (props: UserProps) => {
                     placeholder="Qualification of the Doctor"
                   />
 
-                  <MonthFormField
+                  <TextFormField
                     {...field("doctor_experience_commenced_on")}
                     required
```

Comment:
👀 
![image](https://user-images.githubusercontent.com/25143503/229008104-f86bca1d-2cc4-43b3-a1e1-c50e930c7262.png)

```suggestion
                    required
                    min={0}
```

Reply:
😬